sigpy/mri/samp.py

```diff
@@ -220,6 +220,9 @@ def _poisson(nx, ny, max_attempts, radius_x, radius_y, calib, seed=None):
                         ):
                             done = False
                             break
+                    else:
```

Comment:
breaks out of double for loop, if one point is found. No need, to go through the others anymore